website/src/components/Banner/index.tsx

```diff
@@ -0,0 +1,36 @@
+"use client";
+import { XMarkIcon } from "@heroicons/react/20/solid";
+import { useEffect } from "react";
+import { initFlowbite } from "flowbite";
+
+export default function Banner({
+  active,
+  children,
+}: {
+  active: boolean;
+  children: React.ReactNode;
+}) {
+  useEffect(() => {
+    // Manually init flowbite's data-toggle listeners since we're using custom components
+    initFlowbite();
+  }, []);
+
+  if (!active) return null;
+
+  return (
+    <div
+      id="banner"
+      tabIndex={-1}
+      className="flex top-14 fixed z-50 gap-8 justify-between items-start py-2 px-4 w-full bg-primary-450 shadow-lg sm:items-center dark:border-neutral-700 dark:bg-neutral-800"
```

Comment:
Should we make it not fixed but just a div after the header so that it doesn't go over the hero section title on smaller screens?

Reply:
Hm, yes we could. Then it scrolls out of view. The dismissal action is only sticky for the duration of your session on the site, so I could see how this would make view docs annoying.

I made this configurable and set it to `fixed` by default.

Reply:
Should scroll out of view and not cover anything up now.